Test/Test.csproj

```diff
@@ -32,12 +32,12 @@
     <ConsolePause>false</ConsolePause>

```

Comment:
Can you explain the implications of this change? It looks like you are removing the log4net.dll file which would make the solution unbuildable on the desktop without using nuget. I'm going to see about install Visual Studio for Mac so I can more easily test here but what are the steps to build after this change? Does the user have to do anything to get the log4net.dll to build and run tests?

Reply:
In the current version, PacketDotNet project uses log4net from NuGet package. The test project used log4net.dll from the git repository that was in a different version. On my VS 2017, W10, I was unable to run tests in Debug configuration because of the version mismatch.

That same should be for nunit.framework.dll and SharpPcap.dll. Both have NuGet packages so I do not see the reason why they should be present in source control. I would say that it is commonly a bad practice to store dlls with code if it can be helped.

Or, SharpPcap and PacketDotNet can be built together for development purposes because of their circular dependencies. I used this setup for previous changes in the huge pull request. This way, breaking change refactorization can safely modify both projects.

My colleague tested it on Visual Studio for Mac and built it ok. I think that NuGet is a standard thing and I am not aware of a building environment that would not have it.

Reply:
@pluskal I wonder what the implications would be for Linux users if monodevelop will fetch the packages. We'll have to consider this further before continuing.

You mention that your colleague was building under VS for MAC. I'm trying to build it here and with the latest master from last night I'm not able to build the tests due to a missing .net451 PacketDotNet.dll file. I'm not seeing any option to build a .net 451 version of PacketDotNet. I'd appreciate some help if you could provide it as I'd like to run and test here under Mac.

Reply:
Good point, I have not tested it with mono. It would be nice to have build environment that would test all combinations. 

I am a bit noobie in multi-platform development. But, my last project in Monodevelop had no issues with NuGet as far as I remember. I'll try to ask my colleague, to try it, but he is on vacation so I am not sure how fast I will know. 

My VS on W10 builds both versions net451 and netstandard2 when building solution. The issue I had was with tests and wrong version of log4net as it was provided with NuGet in different version. Let's try all building envs and see if the NuGet works everywhere.